tasks/RedHat.yml

```diff
@@ -121,3 +121,38 @@
     - zabbix-proxy
     - init
     - database
+
+- name: "RedHat | Install related SELinux package to fix issues"
+  yum:
+    name:
+      - policycoreutils-python
+    state: present
+  register: zabbix_server_dependencies_installed
+  until: zabbix_server_dependencies_installed is succeeded
+  when:
+    - ansible_distribution_major_version == "7" or ansible_distribution_major_version == "6"
+    - zabbix_selinux | bool
+  tags:
+    - zabbix-server
```

Comment:
The tags contains `zabbix-server`, can you change that to `zabbix-proxy`?

Reply:
yes